readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: xwp
 Tags: wp stream, stream, activity, logs, track
 Requires at least: 4.5
 Tested up to: 6.3
-Stable tag: 3.10.0
+Stable tag: 3.10.1

```

Comment:
<!-- simple tasks that must be done -->
**🧹 chore:** Let's not bump the version number just yet. This update will be part of `4.0.1` release and we're going to set the plugin version there, just before releasing.

```suggestion
Stable tag: 3.10.0
```